src/index.js

```diff
@@ -55,7 +55,14 @@ const renderDoc = async function renderDocs(path) {
     const response = await exlClient.getArticle(id);
     const md = response.data.FullBody;
     const meta = response.data.FullMeta;
-    const { convertedHtml, originalHtml } = await md2html(md, meta);
+    const lastUpdated = response.data.UpdatedUTC;
```

Comment:
@devisree29 I suppose we should pick these from `response.data.FullMeta`. 
`\nlast-update: 2023-09-22\nlevel: Beginner,Intermediate,Experienced\n`
Values coming in `response.data.UpdatedUTC` is probably not right. 
Reference:
https://experienceleague-dev.corp.adobe.com/docs/journey-optimizer-learn/tutorials/introduction-to-journey-optimizer/mobile-capabilities-for-developers.html?lang=en
https://experienceleague.adobe.com/api/articles/recGseE0j7HEteLlx

Reply:
Updated the code and removed lastupdate and level from the required files and changed the logic to extract all the data from meta tag @Sivaramvt